src/display/api.js

```diff
@@ -1088,16 +1088,24 @@ var PDFWorker = (function PDFWorkerClosure() {
   var nextFakeWorkerId = 0;
 
   function getWorkerSrc() {
+    var suffix;
```

Comment:
I have removed them following the instructions of @yurydelendik  and using a new regex that will take care of everything.